run-wasm/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "run-wasm"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cargo-run-wasm = { git = "https://github.com/rukai/cargo-run-wasm" } # TODO: use crates.io release before merging
```

Comment:
We technically don't need a crates.io release before using this in an xtask.

Reply:
technically yes, may as well do the release before hand though.